config/routes.rb

```diff
@@ -1,2 +1,11 @@
 Rails.application.routes.draw do
+
+  root "application#welcome"
```

Comment:
Consider creating a "WelcomeController#show` instead, which will follow the RESTful convention.

Also, another convention is to place the default root path as the last declaration in the routes file.